Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.8
+MAINTAINER Rafael Bodill <justrafi@gmail.com>
```

Comment:
`MAINTAINER` is deprecated. Please use `LABEL maintainer="Rafael Bodill <justrafi@gmail.com>"` instead.